kafka-ui-react-app/src/components/common/NewTable/LinkTopic.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const LinkTopic = ({ row, to }: any) => {
```

Comment:
let's call it LinkCell and update all the existing tables with same link components